osbs/build/build_request.py

```diff
@@ -732,6 +737,11 @@ def render(self, validate=True):
             self.spec.validate()
         super(ProductionBuild, self).render()
 
+        repo_name = git_repo_humanish_part_from_uri(self.spec.git_uri.value)
+        # What about forked repos?
```

Comment:
In the case of a forked repository, the repo_name will be the same. The only way to distinguish them would be to store the whole git uri as a label as opposed to just the repository name.